fastlane/lanes/screenshots.rb

```diff
@@ -92,6 +87,8 @@
   #####################################################################################
   desc "Downloads translated promo strings from the translation system"
   lane :download_promo_strings do |options|
+    # FIXME: Make this app dependent instead (WordPress vs Jetpack)
+
```

Comment:
I decided to keep this part for a future subsequent PR, as I'm still undecided if it's still useful to keep this lane, dedicated to download only `play_store_screenshot_*` keys separately here… as opposed to just use the existing `download_metadata_strings` which already does the same.

Besides, currently this `download_promo_strings` lane is still flawed by the bug of us relying on `source_locale: 'en-US'` to get the original English copy from GlotPress' source, instead of copying the local `.txt` file that was used as source of truth to generate the `.po` file.

So between having this lane download the screenshot txt files in a different folder (namely `playstoreres`) separate than the rest of the metadata, this flaw of `source_locale`, and the fact that it kinda duplicates what `download_promo_strings`, I'm considering deleting this lane and relying on `download_promo_strings` instead; but that would require some additional changes to go with that, especially updating the `fastlane/screenshots.json`  file to reference the `.txt` files via the `metadata/android/%s/…` path instead of `playstoreres/metadata/%s/…`, etc, so I figured this would lead to bigger changes and was best for a separate PR.

Reply:
I assume there was a typo in this sentence, where `download_promo_strings` sneaked in instead of  `download_metadata_strings`, is that right?

> and the fact that it kinda duplicates what `download_promo_strings`, I'm considering deleting this lane and relying on `download_promo_strings` instead;

I like the idea of relying solely on `download_metadata_strings`.

One concern is that it might generate some noise if one of the strings in `download_promo_strings` is changed at a time when we are not interested in generating new screenshots, but that possibility seems worth the additional efficiency of only having to call GlotPress once per locale as well as having less code to maintain here.

Reply:
Yes this was meant to be "and relying on ~`download_promo_strings`~ `download_metadata_strings` instead;" indeed. Nice catch!